### ISSUE TITLE
Elements must have sufficient color contrast (.k-pager-info)

### ISSUE THREAD

**Issue by sryeluri** [telerik/kendo-themes#587]
Below accessibility issue that needs investigation.

Issue Details
Elements must have sufficient color contrast
color-contrast

Element Path
.k-pager-info

Snippet
<span class="k-pager-info k-label">1 - 10 of 14 items</span>

Expected behaviour:
- Element has insufficient color contrast of 2.32 (foreground color: #aaaaaa, background color: #ffffff, font size: 11.3pt, font weight: normal). Expected contrast ratio of 4.5:1

To reproduce
Use K4W to investigate the issue details

Environment
Chrome version 70.0.3538.77

**Comment by sryeluri** [comment]
> @sryeluri which theme is used in this case ?

we are using the default theme

**Comment by sryeluri** [comment]
> can confirm that the default theme colors do not cover the WCAG contrast requirements.
> A dedicated "high contrast" color palette will be included to address this issue. I don't have a timeline for this feature ye

Can you please provide a tentative date that this issue can be fixed.

**Comment by tsvetomir** [comment]
@sryeluri I believe that this is a false positive as the actual theme colors are #656565 and #f6f6f6 . My mistake here, I should've double checked the color values. Thanks @TeyaVes for pointing this out.

![image](https://user-images.githubusercontent.com/247917/52780477-e4778380-3052-11e9-9c92-0a6a35125498.png)

**Comment by sryeluri** [comment]
You can repro this in the sample application  - https://runner.telerik.io/fullscreen/oXokIKeS (f/g - #B3B3B3 , b/g - #FFFFFF)
Code - https://dojo.telerik.com/oXokIKeS 
Contrast still shows - **2.1:1** which is not acceptable per our Accessibility standards.

**Comment by tsvetomir** [comment]
Thanks for the runnable sample. It seems like the issue is for the Office 365 Less theme. I'll move the issue to the https://github.com/telerik/kendo-ui-core repo as it does not apply to the Sass themes in this repository. 

For the time being, I suggest switching to a theme with sufficient contrast or [customizing the Less theme](https://docs.telerik.com/kendo-ui/styles-and-layout/appearance-styling#creating-custom-themes).
